tests/microgrid/test_microgrid_api.py

```diff
@@ -104,7 +104,7 @@ def metadata(self) -> Metadata:
             ),
         )
 
-    @mock.patch("grpc.aio.insecure_channel")
+    @mock.patch("grpclib.client.Channel")
```

Comment:
It is related, because the parsing of the URL will now return a `grpc.client.Channel` and before doing that switch, we were using a `grpc.aio.insecure_channel`. So we need to adapt the tests to work with this change.

Reply:
Oh I see, sorry for the noise then